packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -332,6 +334,7 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     this.locator = locator;
     this.deviceManager = deviceManager;
     const isTeamsMeeting = 'meetingLink' in this.locator;
+    this._logger = createClientLogger('communication-react:composite');
```

Comment:
are we open to adding AzureLogger to the composite packlet and using it in our composites?